sdk/src/main/scala/com/horizen/account/websocket/data/WebsocketAccountResponse.scala

```diff
@@ -0,0 +1,11 @@
+package com.horizen.account.websocket.data
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonView}
+import com.fasterxml.jackson.databind.node.ObjectNode
+import com.horizen.serialization.Views
+
+@JsonView (Array (classOf[Views.Default] ) )
+class WebsocketAccountResponse(@JsonProperty("method") val method: String,
```

Comment:
where is `id`field?
We may reuse RpcResponseSuccess class, this one can be deleted.

Reply:
There is no **id** field in the response accordingly to Ethereum docs (https://geth.ethereum.org/docs/interacting-with-geth/rpc/pubsub)

Reply:
Added a sub class of RpcResponseSuccess without the **id** field